examples/experimental/usernet.yaml

```diff
@@ -0,0 +1,15 @@
+# Example to run vz instance with lima usernet enabled
+vmType: "vz"
+images:
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-amd64.img"
+  arch: "x86_64"
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-arm64.img"
+  arch: "aarch64"
+
+mounts:
+- location: "~"
+- location: "/tmp/lima"
+  writable: true
+mountType: "virtiofs"
+networks:
+- lima: default
```

Comment:
If the new network is stable enough to make it default, the `default` network should be prepended to the network list by default.
If not, the network shouldn't be called "default" at this moment.

Reply:
On my tests, the network looked stable only. But would like to see others opinions as well mainly with to high load and sleep and wake scenarios.

shall i rename it to `experimental` ?

Reply:
`user-v2` or `user-v2-experimental`
(should correspond to the network driver name)

Reply:
ping 🙏

Reply:
@AkihiroSuda 
Shall I go with user-v2 as most people are okay with that naming ?

Also there is still issues related to network freeze on high load. I have been trying to debug it but still in progress. We can merge this as experimental and analyse that on the side as well.

Reply:
Yes, user-v2 SGTM.
We can rename it later if we want, as long as we keep it experimental.

Reply:
Done